cmd/down.go

```diff
@@ -27,11 +27,6 @@ func down(cmd *cobra.Command, args []string) {
 	//read the harbor compose file
 	var harborCompose = DeserializeHarborCompose(HarborComposeFile)
 
-	_, _, err := Login()
```

Comment:
I guess the scenario I was specifically thinking about was replacing the harborApi code with a shared harbor golang client library.  Wilson and I had talked about sharing a common library between Harbor Compose and the ShipIt CLI.  So if we end up replacing it, with this proposed design, we'll lose the CLI login behavior along with it and will have to add the code back to the cmd layer.